tests/bin/test-setup.sh

```diff
@@ -67,8 +67,10 @@ go get -v github.com/tools/godep
 # cleanup any stale example applications
 rm -rf $DEIS_ROOT/tests/example-*
 
-# generate ssh key if it doesn't already exist
+# generate ssh keys if they don't already exist
 test -e ~/.ssh/$DEIS_TEST_AUTH_KEY || ssh-keygen -t rsa -f ~/.ssh/$DEIS_TEST_AUTH_KEY -N ''
+# TODO: parameterize this key required for keys_test.go?
+test -e ~/.ssh/deiskey || ssh-keygen -q -t rsa -f ~/.ssh/deiskey -N '' -C deiskey
```

Comment:
Because the keys_test.go has it hard-coded for a specific test evidently. I didn't intend to fix everything about the tests in this PR, this just moved a hidden requirement noticed by @smothiki out into the open.